src/main/java/com/sparkplatform/api/models/Listing.java

```diff
@@ -357,10 +356,10 @@ public String getSubdivisionName() {
 		public void setSubdivisionName( String subdivisionName ) {
 			this.subdivisionName = subdivisionName;
 		}
-		public String getmLSAreaMinor() {
+		public String getMLSAreaMinor() {
 			return mLSAreaMinor;
 		}
-		public void setmLSAreaMinor( String mLSAreaMinor ) {
+		public void setMLSAreaMinor( String mLSAreaMinor ) {
```

Comment:
> This may have been a copy-and-paste mistake in the first place.

Almost certainly the case... actually even more likely Eclipse auto-generated it this way. :+1: